README.md

```diff
@@ -20,16 +20,19 @@ Provider for Tinkerbell.
   and Tinkerbell
 * It simplifies Kubernetes cluster management using Tinkerbell as underline
   infrastructure provider
-* Create, update, delete Kubernetes project in a declarative fashion.
+* Create, update, delete Kubernetes clusters in a declarative fashion.
 
 ## Current state
 
 7th December 2020 marks the first commit for this project, it starts as a
 porting from CAPP (cluster api provider packet).
 
-As it is now it does not do anything useful. It starts the infrastructure
-manager. Just a go binary with all the boilerplate code and controllers
-bootstrapped.
+As it is now it is able to bootstrap single Control Plane instance clusters using hardware
+managed by Tinkerbell. Support for upgrades and multi-instance Control Plane clusters is
+still outstanding.
```

Comment:
Updated to be more reflective of the current state of the project, and link to the development environment docs

Reply:
Is it worth to mention what people should expect from `v0.1.0`?

Reply:
I was mostly planning on doing the release after getting this PR sorted out, which would allow for cleaning up some of the current docs around creating a template and deploying.

I think the HA support is likely to be a bit of a longer tail and may be easier to implement along with CAPI v1alpha4 (assuming the load balancer provider proposal makes it in)